parsing/jbuild

```diff
@@ -0,0 +1,29 @@
+(jbuild_version 1)
+
+(library
+ ((name parsing)
+  (public_name coq.parsing)
+  (wrapped false)
+  (modules (:standard pcoq g_prim g_constr g_proofs g_vernac))
+  (flags (-rectypes))
+  (libraries (proofs))))
+
+(rule
+ ((targets (g_prim.ml))
+  (deps (g_prim.ml4 ../grammar/grammar.cma))
+  (action (system "camlp5o pa_extend.cmo q_MLast.cmo pa_macro.cmo pr_o.cmo ../grammar/grammar.cma -loc loc -impl ${<} -o ${@}"))))
```

Comment:
you can use `${<}` to include the deps in your action (also drop the system, if possible).